src/app/modules/main/chat_section/module.nim

```diff
@@ -583,6 +583,9 @@ method addNewChat*(
     chatImage = chatDto.icon
 
   var amIChatAdmin = self.amIMarkedAsAdminUser(chatDto.members)
+  if len(chatDto.communityId) != 0:
```

Comment:
Just an additional condition, so that we get the community only if needed
```suggestion
  if not amIChatAdmin and len(chatDto.communityId) != 0:
```